src/Renderer.tsx

```diff
@@ -24,12 +24,15 @@ import {
     VideoNode,
 } from '@prezly/story-content-format';
 import React, { ComponentType, ReactNode } from 'react';
+import ResizeObserver from 'resize-observer-polyfill';
 
 import * as Elements from './elements';
 import { applyTransformations } from './lib';
 import * as Transformations from './transformations';
 import type { Transformation } from './types';
 import { Component, Selector } from './selector';
+window.ResizeObserver = ResizeObserver;
```

Comment:
Just wondering if this is the best way of doing that.

Reply:
1. I'd better move this to `index.ts`, as this is a global top-level action IMO.
2. I'd make it override `window.ResizeObserver` only if it's undefined to make it clear we prefer the native implementation. Technically, that's already how `resize-observer-polyfill` works, but I just want to make it super-clear.

```ts
window.ResizeObserver = window.ResizeObserver ?? ResizeObserver;
```

Reply:
Yeah, I think it makes sense.

Reply:
Hmm, I've checked implementation and I think `resize-observer-polyfill` does this
```
window.ResizeObserver = window.ResizeObserver ?? ResizeObserver;
```
for us already.
```
export default (() => {
    // Export existing implementation if available.
    if (typeof global.ResizeObserver !== 'undefined') {
        return global.ResizeObserver;
    }

    return ResizeObserverPolyfill;
})();
```

Reply:
Yes, I've mentioned this in my comment :) 

> Technically, that's already how resize-observer-polyfill works, but I just want to make it super-clear.

Reply:
I don't know how I missed that... sorry :)